crates/ruff_linter/src/rules/flake8_gettext/mod.rs

```diff
@@ -7,7 +7,7 @@ pub mod settings;
 /// Returns true if the [`Expr`] is an internationalization function call.
 pub(crate) fn is_gettext_func_call(func: &Expr, functions_names: &[String]) -> bool {
     if let Expr::Name(ast::ExprName { id, .. }) = func {
-        functions_names.contains(id)
+        functions_names.contains(&id.to_string())
```

Comment:
We should either change our settings types to use `SmolStr` or (less work) use iterations to avoid these allocations:

```rust
functions_names
    .iter()
    .any(|function_name| function_name == id)
```